src/webxdc.rs

```diff
@@ -404,6 +405,11 @@ impl Context {
             )
             .await?;
 
+        self.emit_event(EventType::WebxdcUpdateStateChanged {
+            msg_id: instance.id,
+            is_sending: true,
+        });
```

Comment:
at the time this event is emitted, get_busy_webxdc_instances() will not include the sending instance. is that on purpose? otherwise, consider to move that after altering `smtp_status_updates`.